umbraco/Umbraco-CMS
# Only add non default version to operation id

# Summary

This revert the changes that adds verison to all operation Ids.
Insted it is only added if it the not default (1.0)

### Test
- Ensure operationIds do not include 1.0, when reading the openApi.
- Add a version two of a file, and ensure those have 2.0 added, so operationIds are still unique.
  - Note that you will need to have `[ApiVersion("1.0")]` on the controllers themselves as this apparently can't be inherited from the base.
  - On the class that also includes another version. Also add `[ApiVersion("2.0")]` and the `[MapToApiVersion("2.0")]` on the new method.
  - Due to this, it can be easier to test on the delivery API, as there is not as many you need APIs to add `[ApiVersion("1.0")]` to
